src/plugins/httpUpload/HttpUploadPlugin.ts

```diff
@@ -227,6 +227,7 @@ export default class HttpUploadPlugin extends AbstractPlugin {
       let element = $(stanza);
       let bodyElement = element.find('html body[xmlns="' + Strophe.NS.XHTML + '"]').first();
       let dataElement = element.find('data[xmlns="urn:xmpp:bob"]');
+      let body = element.find('body');
```

Comment:
That's to unspecific.

Reply:
now using element.children('body') to get the body tag...

Reply:
There can also be multiple body children in XMPP. Just use the `message.getPlaintext()` method.